app-i18n/man-pages-pl/man-pages-pl-0.7.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2017 Gentoo Foundation
```

Comment:
Your calendar seems to be off.

Plus, please squash commits and format commit messages according to GLEP 66, in particular prefix them with `cat/pn:`.